Dockerfile

```diff
@@ -1,12 +1,18 @@
 FROM jupyter/datascience-notebook
 
-MAINTAINER Pierre Bellec <pierre.bellec@gmail.com>
+LABEL maintainer="Pierre Bellec <pierre.bellec@gmail.com>"
 
 USER jovyan
 
-RUN pip install nilearn
-
-# Get the files of the project
+# Copying the repository inside the container
 RUN wget https://github.com/brainhack101/introML/archive/0.1.zip
 RUN unzip 0.1.zip
 RUN rm 0.1.zip
+
+COPY . /home/jovyan 
+
+# Instaling the Kamalaker's main fecther
+RUN pip install --user git+https://github.com/KamalakerDadi/nilearn.git@16f2df26401e5b6a16ff134f4da71ff920e3ac40
+
+# Downloading the data
+RUN ["/bin/bash", "/home/jovyan/data_fetch.sh"]
```

Comment:
How big is the resulting container if you're fetching both the atlas and the images ?

Reply:
It is big... 7.5GB
But for now let's try this and I will talk with felix-antoine to cache the data on calcul-quebec somehow

Reply:
although this may be a poisonous gift, it may make more sense to list @illdopejake or @emdupre as maintainers...

Reply:
I think @illdopejake and I are happy to see you as the maintainer, @pbellec !

Especially for listing in the Dockerfile, I imagine that the issues springing up there will likely be around the environment and data. Not to mention that you could handle the content, too :) Up to you if you'd like to accept that role, of course !

Reply:
OK so happy to be the maintainer, for better or for worst :)